roles/ipareplica/library/ipareplica_prepare.py

```diff
@@ -168,6 +168,10 @@
     description: Do not configure OpenSSH server
     type: bool
     required: no
+  subid:
```

Comment:
Is the ipaclient option used in ipareplica role and ipareplica_prepare?

Reply:
As far as I understood, the `subid` option is used in ipareplica. In the comments to commits [1](https://pagure.io/freeipa/c/571b6b81c3e79ddc89db07e99b12b3352294cffd) and [2](https://pagure.io/freeipa/c/49ab92c5ef067e67ebe3d53711b027465c591760)